sample/.azureFunctions/swagger/swagger.json

```diff
@@ -0,0 +1,1046 @@
+﻿{
+    "swagger": "2.0",
+    "info": {
```

Comment:
This was generated using your new endpoint, and the idea is that we'll keep it generally up to date?